awx_collection/plugins/modules/tower_inventory_source_update.py

```diff
@@ -88,8 +90,8 @@
 def main():
     # Any additional arguments that are not fields of the item can be added here
     argument_spec = dict(
+        name=dict(required=True),
```

Comment:
This needs to have the alias info in it, like so:
`name=dict(required=True, aliases=['inventory_source']),`

Reply:
oh, right, added it to docstring and forgot the arg spec